infrastructure/templates/template.json

```diff
@@ -1426,26 +1430,23 @@
             "DeploymentRepo": "[parameters('deploymentRepo')]",
             "DeploymentScript": "[parameters('deploymentScript')]"
           },
-          "apiVersion": "2020-08-01-preview",
+          "apiVersion": "2015-01-01",
           "dependsOn": [
             "[variables('sqlserverName')]"
           ],
-          "sku": {
-            "name": "GP_S_Gen5",
-            "tier": "GeneralPurpose",
-            "family": "Gen5",
-            "capacity": "[parameters('maxCapacityStatisticsDb')]"
-          },
-          "kind": "v12.0,user,vcore,serverless",
           "properties": {
+            "edition": "Standard",
             "collation": "SQL_Latin1_General_CP1_CI_AS",
-            "maxSizeBytes": 268435456000,
-            "catalogCollation": "SQL_Latin1_General_CP1_CI_AS",
-            "zoneRedundant": false,
-            "readScale": "Disabled",
+            "maxSizeBytes": "268435456000",
             "autoPauseDelay": "[parameters('autoPauseDelayStatisticsDb')]",
-            "storageAccountType": "GRS",
             "minCapacity": "[parameters('minCapacityStatisticsDb')]"
+          },
+          "kind": "v12.0,user,vcore,serverless",
```

Comment:
This seems to be introduced when compared with the commit https://github.com/dfe-analytical-services/explore-education-statistics/commit/e141123084f3f1465e740146a566c61436df53e0#diff-df68079c12e13e39423a2e795f489815684f257e5c884a2c15f38cde357671e8 and also the properties above are different, like `requestedServiceObjectiveName` is not reinstated.